azerothcore/azerothcore-wotlk
# fix(DB/quest_template): Fix correct text

https://tbc.wowhead.com/quest=2459/ferocitas-the-dream-eater
https://tbc.wowhead.com/quest=287/frostmane-hold
https://tbc.wowhead.com/quest=317/stocking-jetsteam



## Changes Proposed:
-  Fix correct text for this quest 
https://tbc.wowhead.com/quest=2459/ferocitas-the-dream-eater
https://tbc.wowhead.com/quest=287/frostmane-hold
https://tbc.wowhead.com/quest=317/stocking-jetsteam

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/1696
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7824

## SOURCE:


## Tests Performed:

- Ran SQL, 11 Affected rows, 0 errors
![image](https://user-images.githubusercontent.com/44707108/132985010-992b9ac0-dd8f-4f18-b7ba-cc41f4cec083.png)

- Test in-game
![image](https://user-images.githubusercontent.com/44707108/132985202-1bebdf72-bea6-4efb-b82a-38acb0bd78ce.png)


## How to Test the Changes:


1. .q add 2459 / 289 / 317
2. .q complete 2459 / 289 / 317
3. check quest tracker



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
